inspector/src/textureHelper.ts

```diff
@@ -68,7 +68,7 @@ export class TextureHelper {
         if (rttInternalTexture && internalTexture) {
             const samplingMode = internalTexture.samplingMode;
             texture.updateSamplingMode(Texture.NEAREST_NEAREST_MIPNEAREST);
-            scene.postProcessManager.directRender([lodPostProcess], rttInternalTexture);
+            scene.postProcessManager.directRender([lodPostProcess], rtt.renderTarget);

```

Comment:
in this case why is the code looking for rttInternalTexture in the previous if ? it is a bit confusing to me ?